doc/api/zlib.md

```diff
@@ -1033,6 +1032,7 @@ Decompress a chunk of data with [`Unzip`][].
 [`DataView`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView
 [`DeflateRaw`]: #zlib_class_zlib_deflateraw
 [`Deflate`]: #zlib_class_zlib_deflate
+[`deflateInit2` and `inflateInit2`]: https://zlib.net/manual.html#Advanced
```

Comment:
It seems this needs to be placed after the `` [`Unzip`] `` since ASCII sorting means upper case goes first?